docs/sources/live/configuration.md

```diff
@@ -0,0 +1,113 @@
++++
+title = "Configuration"
+description = "Grafana Live configuration guide"
+keywords = ["Grafana", "live", "guide", "websocket"]
+weight = 120
++++
+
+# Grafana Live configuration guide
+
+Grafana Live is enabled by default, but as of Grafana v8 it has a strict default for maximum number of connections per Grafana server instance. 
+
+## Max number of connections
+
+Grafana Live uses persistent connections (WebSocket at the moment) to deliver real-time updates to clients. You may need to tune your server infrastructure a bit.
+
+For this reason the number of maximum connections users can establish with Grafana is limited to 100:
+
+```
+[live]
+# max_connections to Grafana Live WebSocket endpoint per Grafana server instance. See Grafana Live docs if you are
+# going to make it higher than default 100 since this can require some OS and infrastructure tuning.
+max_connections = 100
+```
+
+Before making this limit higher make sure your server and infrastructure allow handling more connections. Below we collected a number of common problems which could happen when managing persistent connections, in particular WebSocket connections.
+
+### WebSocket
+
+Grafana Live uses WebSocket protocol to deliver real-time updates to a frontend application. WebSocket is a persistent connection which starts with HTTP Upgrade request (using the same HTTP port as the rest of Grafana) and then switches to a TCP mode where WebSocket frames can travel in both directions between a client and a server.
+
+Each logged-in user opens a WebSocket connection – one per browser tab.
+
+Introducing a persistent connection leads to some things Grafana users should be aware of.
+
+### Resource usage
+
+Each persistent connection will cost some memory on a server. Typically, this should be about 50 KB per connection at this moment. Thus a server with 1 GB RAM is expected to handle about 20k connections max. Each active connection consumes an additional CPU resources since client and server send PING/PONG frames to each other to maintain a connection.
+
+Of course, it will cost additional CPU as soon as streaming functionality is used. The exact CPU resource utilization is hard to estimate since it heavily depends on Grafana Live usage pattern.
```

Comment:
```suggestion
Each persistent connection costs some memory on a server. Typically, this should be about 50 KB per connection at this moment. Thus a server with 1 GB RAM is expected to handle about 20k connections max. Each active connection consumes additional CPU resources since the client and server send PING/PONG frames to each other to maintain a connection.

Using the streaming functionality results in additional CPU usage. The exact CPU resource utilization can be hard to estimate as it heavily depends on the Grafana Live usage pattern.
```